### DISCUSSION TITLE
DiffusionUniformKh diffusivity parameter

### DISCUSSION THREAD

**Discussion by emma97956461** [Parcels-code/Parcels#1167] [Q&A]
Hi! I'm pretty new to OceanParcels and Python in general, and I'd be grateful for any input on my issue!
I'm using OceanParcels and implementing in sequence an advection kernel and the DiffusionUniformKh kernel. What I lack to understand, even after reading the literature behind this, is what the diffusivity parameter exactly is.  When I put as an input, for example, "diffusivity":20, am I imposing that the parcels are diffusing 20 meters per second in all directions? What are the units of my input?
Thank you all any help!

**Comment by erikvansebille** [comment]
Dear @emma97956461; the units for the `Kh` fields are in m<sup>2</sup>/s. See also the Unit Conversion tutorial at https://nbviewer.org/github/OceanParcels/parcels/blob/master/parcels/examples/tutorial_unitconverters.ipynb for more info
